features/upgrade/machines/machines.feature

```diff
@@ -1,6 +1,11 @@
 Feature: Machine-api components upgrade tests
+  @upgrade-prepare
+  Scenario Outline: Cluster operator should be available after upgrade - prepare
+    Given the expression should be true> "True" == "True"
```

Comment:
@liangxia can you please explain what this check is for?

Reply:
@pruan-rht  Act as a place-holder.

Reply:
@liangxia placeholder for what?

Reply:
@pruan-rht  According to our upgrade workflow, we need an upgrade-prepare and upgrade-check for each scenario. But some of them do not need any prepare steps, which lead to errors "can not find scenarios" in the log, so we just add a simple/useless step here to get rid of the errors in the log.

Reply:
ok, I see, perhaps we should add this note into the PR

Reply:
@pruan-rht  Added the note into the PR.